zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -325,13 +368,21 @@ where
             // Convert into TransactionTemplates
             let mempool_txs = mempool_txs.iter().map(Into::into).collect();
 
-            let empty_string = String::from("");
+            let mut mutable: Vec<String> = constants::GET_BLOCK_TEMPLATE_MUTABLE_FIELD.iter().map(ToString::to_string).collect();
+
+            // If we are in the testnet and received expected difficulty equals to the target difficulty limit for the Testnet then we
+            // are mining a minimum difficulty block.
```

Comment:
I am not sure if this is enough to apply the change.

Reply:
It's possible that `cur_time` is before the 15 minute limit, but `max_time` is after it.

So it's almost right, but we need to check if the difficulty for `max_time` is the minimum difficulty. (`chain_info.expected_difficulty` is the difficulty for `cur_time`.)

Since we're moving all the time stuff to the state anyway, we can do a difficulty check on `max_time` there?

Reply:
Did you want me to make this change?

Reply:
Fixed by a22e3b5